teensy/main/main.ino

```diff
@@ -140,5 +176,22 @@ void print_all_sensors() {
   Serial.println(count);
   count++;
   Serial1.println();
+  
+  output.concat("^Count: ");
+  output.concat(count);
+
+  myFile = SD.open("SMV.txt", FILE_WRITE);
```

Comment:
I'm assuming with this line that the Teensy is writing to a file called "SMV.txt" every time. Could we change this to the trial name so that we don't overwrite previous data?

Reply:
Do you want each test to be saved in a new text file? Maybe if I name it something like trial.txt it will create trial.txt the first time and then trial(1).txt and trial(2).txt and so on. I can try this out!

Reply:
I think it'll overwrite the file if you do that. Ideally, we should name the files so that we know exactly when each data file is from. Maybe we can get the time from the ESP or get some time module for the Teensy

Reply:
i'll look into it - pretty sure there is a way to do that!